Services/DataX.Flow/Flow.SchemaInferenceService/PackageRoot/ServiceManifest.xml

```diff
@@ -4,7 +4,7 @@ Copyright (c) Microsoft Corporation.  All rights reserved.
 Licensed under the MIT License
 -->
 <ServiceManifest Name="Flow.SchemaInferenceServicePkg"
-                 Version="1.0.1"
+                 Version="1.0.0"
```

Comment:
why going back to .0?

Reply:
This change actually fixes a bug where the version needs to be consistent across various manifest files. This version exists in various other files as well viz. ApplicationManifest.xml and all these files must have the same version. re: why 1.0.0?, IMO, there is no need to rev up the version. It should not  have been changed to 1.0.1 for partial files. This change fixes the break which blocks the local-cloud deployment of service fabric backend services scenario for developers.